### ISSUE TITLE
Errors importing junit xml reports

### ISSUE THREAD

**Issue by albfan** [reportportal/reportportal#2377]
**Describe the bug**

So far import from junit xml report works correctly.

Based on this format description:

https://github.com/testmoapp/junitxml

There are some info that fails to import or is missing:

- `failure` node ignore the `message` attribute.
- tags `system-out` and `system-err` work correctly under `test-case` (they are shown in log view) but if we use them on `test-suite`, logs are not filled and this error is shown:

> ReportPortalExceptionResolver > Launch 'null' not found. Did you use correct Launch ID?

...

**Expected behavior**

`system-out` and `system-err` allowed on whole junit, test-suite and test-case
`message` on failures to be honored
...

**Screenshots**
No

**Versions:**
 - Linux, Chrome
 - Version of RP [find it under Login form, copy as is]

***Workaround:**

What would be the API to upload logs related with xml report (general log, test suite log, test-case logs)

**Comment by albfan** [comment]
Example junit xml that should work but fails due to `system-err` and  `system-out` under `testsuite` (on testcase works)

> ReportPortalExceptionResolver > Launch 'null' not found. Did you use correct Launch ID?

```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Test run" tests="8" failures="1" errors="1" skipped="1"
    assertions="20" time="16.082687" timestamp="2021-04-02T15:48:23">

    <testsuite name="Tests.Registration" tests="8" failures="1" errors="1" skipped="1"
        assertions="20" time="16.082687" timestamp="2021-04-02T15:48:23"
        file="tests/registration.code">

        <properties>
            <property name="version" value="1.774" />
            <property name="commit" value="ef7bebf" />
            <property name="browser" value="Google Chrome" />
            <property name="ci" value="https://github.com/actions/runs/1234" />
            <property name="config">
                Config line #1
                Config line #2
                Config line #3
            </property>
        </properties>

        <system-out>Data written to standard out.</system-out>

        <system-err>Data written to standard error.</system-err>

        <testcase name="testCase1" classname="Tests.Registration" assertions="2"
            time="2.436" file="tests/registration.code" line="24" />
        <testcase name="testCase2" classname="Tests.Registration" assertions="6"
            time="1.534" file="tests/registration.code" line="62" />
        <testcase name="testCase3" classname="Tests.Registration" assertions="3"
            time="0.822" file="tests/registration.code" line="102" />

        <testcase name="testCase4" classname="Tests.Registration" assertions="0"
            time="0" file="tests/registration.code" line="164">
            <skipped message="Test was skipped." />
        </testcase>

        <testcase name="testCase5" classname="Tests.Registration" assertions="2"
            time="2.902412" file="tests/registration.code" line="202">
            <failure message="Expected value did not match." type="AssertionError">
                <!-- Failure description or stack trace -->
            </failure>
        </testcase>

        <testcase name="testCase6" classname="Tests.Registration" assertions="0"
            time="3.819" file="tests/registration.code" line="235">
            <error message="Division by zero." type="ArithmeticError">
                Error description or stack trace
            </error>
        </testcase>

        <testcase name="testCase7" classname="Tests.Registration" assertions="3"
            time="2.944" file="tests/registration.code" line="287">
            <system-out>Data written to standard out.</system-out>

            <system-err>Data written to standard error.</system-err>
        </testcase>

        <testcase name="testCase8" classname="Tests.Registration" assertions="4"
            time="1.625275" file="tests/registration.code" line="302">
            <properties>
                <property name="priority" value="high" />
                <property name="language" value="english" />
                <property name="author" value="Adrian" />
                <property name="attachment" value="screenshots/dashboard.png" />
                <property name="attachment" value="screenshots/users.png" />
                <property name="description">
                    This text describes the purpose of this test case and provides
                    an overview of what the test does and how it works.
                </property>
            </properties>
        </testcase>
    </testsuite>
</testsuites>
```

**Comment by Pink-Bumblebee** [comment]
@albfan , as DEV said: 
We adhered to the [maven-surefire](https://github.com/apache/maven-surefire/blob/master/maven-surefire-plugin/src/site/resources/xsd/surefire-test-report.xsd) scheme, which does not include logs in the suites.
It is impossible to account for the thousands of variations that users might generate in their reports. For this reason, data import has now been moved to plugins, allowing users to create their own custom plugins. We kindly ask you to review the documentation https://reportportal.io/docs/developers-guides/PluginDevelopersGuide/DevelopACustomImportPlugin/
